data/xml/1956.preconf.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="1956.preconf">
+  <volume id="1" ingest-date="2021-09-29">
+    <meta>
+      <booktitle>Proceedings of the International Conference on Mechanical Translation</booktitle>
+      <address>Massachusetts Institute of Technology</address>
+      <month>20 October</month>
+      <year>1956</year>
+      <url hash="5fa776a3">1956.preconf-1</url>
+    </meta>
```

Comment:
I think the entire volume should be ingested as a PDF volume (under `<meta>`), not as front matter. We should then split up the papers and associate them with the individual entries.